modules/nodes-descheduler-configuring-profiles.adoc

```diff
@@ -35,32 +35,30 @@ spec:
   logLevel: Normal
   managementState: Managed
   operatorLogLevel: Normal
+  mode: Predictive                                     <1>
   profileCustomizations:
-    namespaces:                                        <1>
+    namespaces:                                        <2>
       excluded:
       - my-namespace
-    podLifetime: 48h                                   <2>
-    thresholdPriorityClassName: my-priority-class-name <3>
-  profiles:                                            <4>
+    podLifetime: 48h                                   <3>
+    thresholdPriorityClassName: my-priority-class-name <4>
+  profiles:                                            <5>
   - AffinityAndTaints
-  - TopologyAndDuplicates                              <5>
+  - TopologyAndDuplicates                              <6>
   - LifecycleAndUtilization
   - EvictPodsWithLocalStorage
   - EvictPodsWithPVC
 ----
---
```

Comment:
This needs to be here at the beginning of the callouts and at the end of the callouts, otherwise the important note doesn't indent properly, and the callout numbering restarts afterward

![image](https://user-images.githubusercontent.com/7443887/220410264-ccb6976e-8f24-4c33-b731-bb80064ca34b.png)